packages/node/src/node.ts

```diff
@@ -143,8 +153,10 @@ export class Node extends Base {
       visitedObjects.delete(value);
 
       return logClone;
-    } else {
+    } else if (typeof value === 'undefined') {
       return undefined;
+    } else {
+      return `<omitted unserializable ${typeof value}>`;
```

Comment:
I think that we could call this simply (as a personal preference), but this works too.
```suggestion
      return `<${typeof value}>`;
```

Reply:
I wanted to be consistent with other "we've omitted this because of ..." and to be searchable easily in code.